welcome/index.adoc

```diff
@@ -200,31 +189,25 @@ xref:../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storag
+- **xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Use custom resource definitions (CRDs) to modify the cluster]**: Cluster features implemented with Operators, can be modified with CRDs. Learn to xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition_crd-extending-api-with-crds[create a CRD] and xref:../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[manage resources from CRDs].
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-- **xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]**: Choose from CPU, memory and other system resources to xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas].
+- **xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]**: Choose from CPU, memory, and other system resources to xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas].
 endif::[]
 
-- **xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]**: You can reclaim space by pruning unneeded Operators, groups, deployments, builds, images, registries, and cron jobs.
+- **xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]**: Reclaim space by pruning unneeded Operators, groups, deployments, builds, images, registries, and cron jobs.
 
 - **xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
 - **Update a cluster**:
-To upgrade your {product-title} to a later version, use the Cluster Version Operator (CVO).
-If an update is available from the Container Platform update service, you apply that cluster update from either the xref:../updating/updating-cluster.adoc#updating-cluster[web console] or the xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[CLI].
+Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster. If an update is available from the Container Platform update service, you apply that cluster update from either the {product-title} xref:../updating/updating-cluster.adoc#updating-cluster[web console] or the xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[OpenShift CLI] (`oc`).
```

Comment:
Do we know if "Container Platform update service" could just be "{product-title} update service" here? I see a hit for "OpenShift Container Platform update service" in `updating/installing-update-service.adoc`.

Reply:
Actually, in https://deploy-preview-33427--osdocs.netlify.app/openshift-enterprise/latest/updating/installing-update-service.html (linked in the next bullet) it looks like it refers to "OpenShift Update Service (OSUS)" throughout there, so if these are referring to the same thing, sync up w/ that?